src/navigators/__tests__/DrawerNavigator-test.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+
+import 'react-native';
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import DrawerNavigator from '../DrawerNavigator';
+
+const _getScreen = (label: string) => {
```

Comment:
What is this `label: string` sounds like TypeScript, but in JavaScript. Is it flow?